### PR TITLE
Updated rosettanet release notes for mule 4

### PR DESCRIPTION
**Pull request by alexandruatomei9**

### CODE REVIEW

File: modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -3,15 +3,49 @@ ifndef::env-site,env-github[]
...
+|3C3 Notify of Invoice |v01.00, v01.11
+|===
+
+=== Fixed in this Release
+
+* Now works with Mule Runtime 4.2.0 and Java 11. Added the following missing dependencies: org.glassfish.jaxb:jaxb-jxc:2.3.0, javax.activation:javax.activation-api:1.2.0, javax.xml.bind:jaxb-api:2.3.1, org.glassfish.jaxb:jaxb-runtime:2.3.1, com.sun.xml.bind:jaxb-xjc:2.3.0, javax.xml.bind:jaxb-api:2.3.1, com.sun.xml.bind:jaxb-impl:2.3.1, com.sun.xml.bind:jaxb-core:2.3.0
+
+=== Known Issues
+
+The Mule 4 RosettaNet connectory only supports:
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: Typo: connectory

File: modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc [hunk 2]
```diff
@@ -3,15 +3,49 @@ ifndef::env-site,env-github[]
...
+
+* Not supported on Flow Designer
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: Weird to have the first bullet be Not supported... I would have this as the last thing and probably not as a bullet. Something like: The Anypoint Connector for RosettaNet does not support Flow Designer (or "is not supported on Flow Designer)

File: modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc [hunk 3]
```diff
@@ -3,15 +3,49 @@ ifndef::env-site,env-github[]
...
+|Anypoint Studio |7.1.0 and later
+|PIP 0A1 Notification of Failure |v02.00
+|3A4 Request Purchase Order |v02.02 |
+|3A6 Distribute Order Status |v02.00, v02.02, v02.03
+|3B2 Notify of Advance Shipment |v01.00
+|3C3 Notify of Invoice |v01.00, v01.11
+|===
+
+=== Fixed in this Release
+
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: I think these would be better as a list, but don't know if you want to take the time to do that.

Comment (mcherem) [reply modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: This is the new format for the RNs. You can give Meera feedback on it if you like.

File: modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: This was fixed. Somehow my edits did not get saved. You were looking at the original doc. Darn!

File: modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/rosettanet-connector-release-notes-mule-4.adoc]: Yeah, I rewrote it to have the not supported last and edited one of the sentences. Here's the edited stuff:
The Mule 4 RosettaNet connector only supports:

Asynchronous response messages

RNIF 2.00.01

Cluster 3 PIPs (limited selection)

Message signing (optional), not encryption

Certificate revocation managed directly by a user in the keystore

A maximum message size of 5 MB

It is not supported on Flow Designer.

**Pull request merged**